### ISSUE TITLE
SSH protocol doesn't work on Mac

### ISSUE THREAD

**Issue by sw1sh** [ropensci/git2r#270]
Installed latest versions of everything I could.

```
> git2r::libgit2_features()
$threads
[1] FALSE

$https
[1] TRUE

$ssh
[1] TRUE
> git2r::remote_ls('git@github.com:ropensci/git2r.git')
Error in remote_ls("git@github.com:ropensci/git2r.git") :
  ignoring SIGPIPE signal
```

Or it just hangs forever with no output.

**Comment by achubaty** [comment]
I have the same issue, which causes `devtools::install_github()` to hang forever.

**Comment by achubaty** [comment]
@sw1sh I can get SSH working if I manually specify the key to use:

```r
cred <- cred_ssh_key(publickey = "~/.ssh/id_rsa_GitHub.pub",
                     privatekey = "~/.ssh/id_rsa_GitHub",
                     passphrase = character(0))
remote_ls('git@github.com:ropensci/git2r.git', credentials = cred)
```

does that work for you? (unfortunately, `devtools` won't let me pass the `credentials` argument, so it doesn't workaround my issue)

**Comment by stewid** [comment]
Fixed in e289c233968d130098ca0bb68925f30244d297d3
